docs/tutorials/ratelimit.md

```diff
@@ -59,6 +59,15 @@ header value. As an example for Authorization Header you would use:
 clientRatelimit(10, "1m", "Authorization")
 ```
 
+The optional third argument can create an AND combined Header
+ratelimit, if you use `,` to separate the them. For example all of the
```

Comment:
"ratelimit, if you use `,` to separate the them" -> "ratelimit. The headers names must be separated by `,`."

Reply:
The headers names -> The header names 

not ?

Reply:
fixed